lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -202,7 +203,7 @@ class window.Turbolinks
       triggerEvent 'page:change'
       triggerEvent 'page:update'
 
-    return
+    return [].slice.call(body.children)
```

Comment:
New behaviour here, who consumes this?  Is anyone relying on `undefined` return value?

What follows is general musing and not specifically related, but from what I can glean from re-reading this path, it seems we:

1. `refreshNodes(getNodesWithRefreshAlways(), body)` : old document has `refresh-always` nodes from new document swapped in
2. `persistStaticElements(body)` : we fetch nodes from `static` nodes from old document, then place them in new document's body (makes sense, maintains static-ness)
3. **Either** a.) `refreshAllExceptWithKeys(options.exceptKeys, body) : we refresh a bunch of nodes in new document's body, except a few user marked not to refresh or b.) `deleteRefreshNeverNodes(body) : we remove `refresh-never` nodes from new document body
4. Finally, we just replace old document's body with new document's body.

Step 4 makes step 1 seem redundant; why modify old document body if we're just going to replace it later? Perhaps step 1 is only useful for listeners listening for the event `page:before-partial-replace` (called within step1)?  It's tricky to even get a good feel for specifically what's happening to which document from these lines.  Think I should open an issue to refactor and make this more clear?

In any case, I feel like the return value from this function is meant to represent "only the list of nodes that have changed", but with the concept of static nodes persisted from old body potentially being within the children, I guess I'm at a weird situation where, "technically accurate" but maybe a bit misleading?  `body.children` could include `tg-static` nodes right?  But I guess more weirdly, `body.children` could be all new nodes, but some of their descendants could be old nodes.  Not sure if we care to make that distinction.  Maybe why we never returned anything before?